From felipe.balbi@nokia.com  Mon Dec  7 16:43:42 2009
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Fri,  4 Dec 2009 15:47:46 +0200
Subject: USB: musb: musb_gadget: fix sparse warning
To: linux-usb@vger.kernel.org
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Felipe Balbi <felipe.balbi@nokia.com>
Message-ID: <1259934467-14597-5-git-send-email-felipe.balbi@nokia.com>


Fix the following sparse warnings:

drivers/usb/musb/musb_gadget.c:1161:5: warning: symbol 'musb_gadget_set_halt' was not declared. Should it be static?

drivers/usb/musb/musb_gadget.c:1244:5: warning: symbol 'musb_gadget_set_wedge' was not declared. Should it be static?

Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/musb_gadget.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/musb/musb_gadget.c
+++ b/drivers/usb/musb/musb_gadget.c
@@ -1211,7 +1211,7 @@ done:
  *
  * exported to ep0 code
  */
-int musb_gadget_set_halt(struct usb_ep *ep, int value)
+static int musb_gadget_set_halt(struct usb_ep *ep, int value)
 {
 	struct musb_ep		*musb_ep = to_musb_ep(ep);
 	u8			epnum = musb_ep->current_epnum;
@@ -1296,7 +1296,7 @@ done:
 /*
  * Sets the halt feature with the clear requests ignored
  */
-int musb_gadget_set_wedge(struct usb_ep *ep)
+static int musb_gadget_set_wedge(struct usb_ep *ep)
 {
 	struct musb_ep		*musb_ep = to_musb_ep(ep);
 
